gulpfile.js

```diff
@@ -1097,13 +1097,24 @@ function createGalleryList() {
         fileList.push('!Apps/Sandcastle/gallery/development/**/*.html');
     }
 
+    // On travis, the version is set to something like '1.43.0-branch-name-travisBuildNumber'
+    // We need to extract just the Major.Minor version
+    var majorMinor = packageJson.version.match(/^(.*)\.(.*)\./);
+    var major = majorMinor[1];
+    var minor = Number(majorMinor[2]) - 1; // We want the last release, not current release
+    var tagVersion = major + '.' + minor;
+
+    // Get an array of demos that were added since the last release.
+    // This includes newly staged local demos as well.
+    var newDemos = child_process.execSync('git diff --name-only --diff-filter=A ' + tagVersion + ' Apps/Sandcastle/gallery/*.html').toString().trim().split('\n');
```

Comment:
Does anyone ever need to build Cesium outside of the git repo, like from a zip file?  What happens if this fails?

Reply:
This will throw if it fails (failing the build).  While the answer has always been nebulous, we don't really support building out of the zip file. However, since it was trivial I changed it to catch the error and print out a warning instead of failing.  Should be good to go.